docker/templates/Dockerfile.test-database.m4

```diff
@@ -31,6 +32,10 @@ COPY admin/ admin/
 COPY lib/ lib/
 COPY script/ script/
 COPY t/sql/initial.sql t/sql/
+COPY entities.json entities.json
```

Comment:
In `admin/MBImport.pl`, `lib/MusicBrainz/Server/Constants.pm` is [used](https://github.com/metabrainz/musicbrainz-server/blob/master/admin/MBImport.pl#L36), which further makes use of `entities.json` [here](https://github.com/metabrainz/musicbrainz-server/blob/master/lib/MusicBrainz/Server/Constants.pm#L380).